libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/LogHelper.kt

```diff
@@ -0,0 +1,28 @@
+package breez_sdk_notification
+
+import org.tinylog.kotlin.Logger
+import java.io.File
+
+@Suppress("unused")
+class LogHelper {
```

Comment:
@erdemyerebasmaz  @dangeross seems like it aim to abstract the use of tinylog from the user as implementation details but using this API will override the users logging dir if they use tinylog in their app since tinylog API is static.

Reply:
The application calls configureLogger(logDir) in onMessageReceived() to configure the logging directory
https://github.com/breez/c-breez/blob/68a6cdd61b60c5ae000b4d4a5b9d581fd2dbb0ae/android/app/src/main/kotlin/com/cBreez/client/BreezFcmService.kt#L28-L50

Reply:
I am thinking on an app that already use tiny log and have that configured with some configuration properties.  Seems like we overriding these properties here, don't we?

Reply:
Only if the application calls configureLogger, no?

Reply:
What happens if the application did not do any configuration? I'm thinking of RN applications also, where nothing would be already configured.

If you think its not needed in the SDK and the application should handle this, we can move it back to c-breez.